savedbooks.js

```diff
@@ -0,0 +1,68 @@
+var savedBooks = JSON.parse(localstorage.getItem("savedBooks")); 
```

Comment:
this can also be done on our existing files with firebase

Reply:
yeah i realized that after when I messaged you on slack that we are using firebase